src/libs/antares/study/binding_constraint/BindingConstraintSaver.cpp

```diff
@@ -21,7 +21,9 @@ bool BindingConstraintSaver::saveToEnv(EnvForSaving& env, const BindingConstrain
     env.section->add("filter-year-by-year", datePrecisionIntoString(bindingConstraint->pFilterYearByYear));
     env.section->add("filter-synthesis", datePrecisionIntoString(bindingConstraint->pFilterSynthesis));
 
-    if (!bindingConstraint->group().empty())
+    if (bindingConstraint->group().empty())
```

Comment:
Shouldn't the default group be set when loading ?

Reply:
Yes, although it's rather equivalent

Reply:
Well, if we set the **default** group when saving, this probably means that, at least for some cases (old studies), the group of a constraint is **empty** during the simulation, right ?
Is it a good thing ? I'm not sure it's clean. Any constraint should always belong to a group with a name, shouldn't it ?

Reply:
We will allow "no binding constraint group", see latest change